python/neuroglancer/chunks.py

```diff
@@ -17,13 +17,40 @@
 import numpy as np
 from PIL import Image
 
+def encode_jpeg(subvol):
+    shape = subvol.shape
+    reshaped = subvol.reshape(shape[0] * shape[1], shape[2])
+    img = Image.fromarray(reshaped)
+    f = io.BytesIO()
+    img.save(f, "JPEG")
+    return f.getvalue()
+
+def decode_jpeg(bytestring, shape=(64,64,64), dtype=np.uint8):
+
+    if len(bytestring) <= 3404: # black jpeg of shape (64,64,64) in mode L
+        return np.zeros(shape=shape, dtype=dtype)
+
+    img = Image.open(io.BytesIO(bytestring))
+
+    # This is how simple it should be but sometimes the data are buggy:
+    # return np.array(img.getdata(), dtype=dtype).reshape(shape).T
+
+    data = np.array(img.getdata(), dtype=dtype)
+
+    if len(data) == 64 ** 3:
```

Comment:
My feeling is that we should try to figure out where the bug in ingestion is. We can then fix any dataset by ingesting from scratch. Otherwise this will cover that we have a bug somewhere else and it will take longer to fix the root cause.